certified-connectors/Witivio/intro.md

```diff
@@ -0,0 +1,9 @@
+Witivio is a powerful chatbot development platform that allows you to simply build chatbots that help you to communicate and engage with your colleagues and employees. No coding necessary. Just install it and you’re ready to go!
```

Comment:
@tdechanterac Intro.md is required in the ISV studio which can be removed from PR.